src/main/java/NotificationHRGUI.java

```diff
@@ -24,43 +25,65 @@ public class NotificationGUI extends JFrame implements ActionListener {
     public JList<String> resolvedNotificationList;
 
 
-    public NotificationGUI() {
+    public NotificationHRGUI(User user) {
```

Comment:
replace User user with int userId (outer layer cannot skip intermediate layers and directly depend on entity)

Reply:
should implement Page interface